Sources/LanguageServerProtocol/SupportTypes/SemanticTokenModifiers.swift

```diff
@@ -47,6 +47,9 @@ public struct SemanticTokenModifiers: OptionSet, Hashable, Sendable {
   public static let fileScope = Self(rawValue: 1 << 19)
   public static let globalScope = Self(rawValue: 1 << 20)
 
+  // The following are our LSP extensions
+  public static let parameterLabel = Self(rawValue: 1 << 21)
```

Comment:
- Could you mark this as `/// **(LSP Extension)**` to be consistent with other LSP extensions we introduce?
- Could you add it to the `LSP Extensions.md` document? 
- I think we should call this `argumentLabel` to be consistent with the notion in swift-syntax.

```suggestion
  /// Argument labels in function definitions and function calls
  ///
  /// **(LSP Extension)**
  public static let argumentLabel = Self(rawValue: 1 << 21)
```